packages/terminology/src/migrations/v1.ts

```diff
@@ -0,0 +1,6 @@
+export const sql = `
+    CREATE TABLE IF NOT EXISTS CodeSystem (
+      id UUID NOT NULL PRIMARY KEY,
+      content TEXT NOT NULL,
```

Comment:
i think SQLITE actually recently added JSONB support. 
would be better to use that then stringify(text)